syntax = "proto3";

// The greeting service definition.
service WordCount {
  // Sends a greeting
  rpc SendText (WordCountRequest) returns (WordCountResponse) {}
}

// The request message containing the user's name.
message WordCountRequest {
  string text = 1;
}

// The response message containing the greetings
message WordCountResponse {
  int32 number_words = 1;
  map<string, int32> words_freq = 2;
}